src/Server/HttpServerConfiguration.php

```diff
@@ -334,6 +337,7 @@ private function validateSetting(string $key, $value): void
                 Assertion::inArray($value, \array_keys(self::SWOOLE_LOG_LEVELS));
 
                 break;
+            case self::SWOOLE_HTTP_SERVER_CONFIG_PACKAGE_MAX_LENGTH:
```

Comment:
I know that this design is really bad, I have it in mind to replace logic example with symfony config component, but for now, this requires separate case.

```suggestion
            case self::SWOOLE_HTTP_SERVER_CONFIG_PACKAGE_MAX_LENGTH:
                Assertion::integer($value, \sprintf('Setting "%s" must be an integer.', $key));
                Assertion::greaterThan($value, 0, 'Package max length value cannot be negative or zero, "%s" provided.');
                
                break;
```

Reply:
I added a separat case block for it